internal/services/compute/windows_virtual_machine_resource.go

```diff
@@ -212,6 +211,20 @@ func resourceWindowsVirtualMachine() *pluginsdk.Resource {
 				}, false),
 			},
 
+			"hotpatching_enabled": {
```

Comment:
pedantic, but shouldn't this be:

```suggestion
			"hot_patching_enabled": {
```

Reply:
Fixed.

Reply:
Service teams specifically requested that the name be "hotpatching_enabled"